terminus-ui/drawer/src/drawer.component.ts

```diff
@@ -131,6 +137,18 @@ export class TsDrawerComponent implements AfterContentChecked, OnDestroy {
    */
   public readonly modeChanged = new Subject();
 
+  /**
```

Comment:
I'll only tag it in one spot since we chatted via slack, but these content children and footer/header margin inputs are all not needed to meet our requirements list.